.github/workflows/universalnrp-test.yml

```diff
@@ -11,11 +15,23 @@ jobs:
       matrix:
         target:
           [
+            { os: centos, version: 8, package-type: RPM, tag: ''},
+            { os: debian, version: 10, package-type: DEB, tag: ''},
+            { os: debian, version: 11, package-type: DEB, tag: ''},
+            { os: mariner, version: 2, package-type: RPM, tag: ''},
+            { os: oraclelinux, version: 8, package-type: RPM, tag: ''},
+            { os: rhel, version: 8, package-type: RPM, tag: ''},
+            { os: rhel, version: 9, package-type: RPM, tag: ''},
+            { os: rockylinux, version: 9, package-type: RPM, tag: ''},
+            { os: sles, version: 15, package-type: RPM, tag: ''},
             { os: ubuntu, version: 20.04, package-type: DEB, tag: ''},
+            { os: ubuntu, version: 22.04, package-type: DEB, tag: ''},
```

Comment:
Should we also add these jobs as required (same as CI unit test jobs)?

Reply:
Eventually yes, some are currently failing  but once they are clean I will introduce them

Reply:
Awesome sounds good!